client/src/components/CompanyProfile.js

```diff
@@ -89,8 +98,20 @@ class CompanyProfile extends React.Component {
     return "/company/manage-profile";
   };
 
-  handleDeleteOpportunity = () => {
-    return "....";
+  ConfirmDelete = id => {
```

Comment:
I am wondering you are trying to delete an opportunity and you adding the functionality to the company profile, what is the relation here, I lost.

Reply:
because when the company is viewing the profile it can see its opportunities and can manage them by editing and deleting

Reply:
Alright, that makes sense now.